shallow_backup/reinstall.py

```diff
@@ -5,25 +5,35 @@
 from .printing import *
 from .compatibility import *
 from .config import get_config
-from shutil import copytree, copyfile
+from pathlib import Path
+from shutil import copytree, copyfile, copy
 
 # NOTE: Naming convention is like this since the CLI flags would otherwise
 #       conflict with the function names.
 
 
-def reinstall_dots_sb(dots_path):
+def reinstall_dots_sb(dots_path,home_path=os.path.expanduser("~")):
 	"""
 	Reinstall all dotfiles and folders by copying them to the home dir.
 	"""
 	empty_backup_dir_check(dots_path, 'dotfile')
 	print_section_header("REINSTALLING DOTFILES", Fore.BLUE)
+	parent = Path(dots_path)
 
-	home_path = os.path.expanduser('~')
 	for file in get_abs_path_subfiles(dots_path):
 		if os.path.isdir(file):
 			copytree(file, home_path, symlinks=True)
 		else:
-			copyfile(file, home_path)
+			son=Path(os.path.dirname(file))
+			destination=""
+			if parent in son.parents:
+				folderLevel = son.relative_to(parent)
+				destination = os.path.join(home_path,folderLevel)
+				if not os.path.exists(os.path.join(home_path,folderLevel)):
```

Comment:
Add spacing after all `,`s in function arguments, and `=`s in variable assignments.